applications/machine_learning/README.rst

```diff
@@ -275,6 +275,28 @@ By default, the application uses the following LED effects:
     * LED blinks with an average frequency if it is connected, but is not actively forwarding data.
     * LED blinks rapidly if it is connected and is actively forwarding data.
 
+Power management
+================
+
+Reducing power consumption is important for every battery-powered device.
+
+In the Machine Learning application specified modules can be suspended or power off there are not in use.
+
+* In the suspended state, the device maintains the active connection.
+* In the powered off state, the CPU is switched to the off mode.
+
+In both cases, most of the functionalities are disabled.
+For example, LEDs and sensors are turned off and advertising is stopped.
+
+Any button press can wake up the device.
+
+For the Thingy:53 sensors supports active power management.
```

Comment:
`sensor`? We only use accelerometer.

Reply:
Supports trigger that can be used for active power management in CAF sensor sampler (to distinguish from Zephyr's power management API calls that are not supported)

Reply:
ok